Tools/CMake/AMReXCUDAOptions.cmake

```diff
@@ -17,16 +17,7 @@ endmacro ()
 #
 message(STATUS "Enabled CUDA options:")
 
-set(AMReX_CUDA_ARCH_DEFAULT "Auto")
-if(DEFINED CMAKE_CUDA_ARCHITECTURES)
-   set(AMReX_CUDA_ARCH_DEFAULT "${CMAKE_CUDA_ARCHITECTURES}")
-endif ()
-if(DEFINED ENV{AMREX_CUDA_ARCH})
-   set(AMReX_CUDA_ARCH_DEFAULT "$ENV{AMREX_CUDA_ARCH}")
```

Comment:
Let us keep that `$ENV{AMREX_CUDA_ARCH}` hint to avoid breakage for users.

If found, we simply set the default for `CMAKE_CUDA_ARCHITECTURES` to it.